### PR TITLE
mon-34

### PR DESCRIPTION
**Pull request by rudolfdeer**

### CODE REVIEW

File: client/App.tsx [hunk 1]
```diff
@@ -1,45 +1,33 @@
...
-    setUser: React.Dispatch<React.SetStateAction<User>>;
-  };
-  Categories: {
-    user: User;
-    setUser: React.Dispatch<React.SetStateAction<User>>;
-  };
-  Intro: {
-    setUser: React.Dispatch<React.SetStateAction<User>>;
-  };
+  Home: undefined;
```
Comment (Alexfario) [inline client/App.tsx:8]: I believe we could omit this type, cause it doesn't help

File: client/App.tsx [hunk 2]
```diff
@@ -1,45 +1,33 @@
...
 
 const Stack = createNativeStackNavigator<StackParamList>();
 
 export default function App() {
-  const [user, setUser] = useState(emptyUser);
 
   return (
     <NavigationContainer>
       <Stack.Navigator>
         <Stack.Screen name="Intro" options={{ headerShown: false }}>
```
Comment (Alexfario) [inline client/App.tsx:20]: I believe we may just add `component={IntroScreen}` and props will be accessible

Comment (Alexfario) [inline client/App.tsx]: I believe we can add `component={IntroScreen}` and props should be accessible

File: client/App.tsx (hunk 1 above)
Comment (rudolfdeer) [reply client/App.tsx:8]: Specifying undefined means that the route doesn't have params. This is from docs: https://reactnavigation.org/docs/typescript/

**Pull request merged**